src/adapters/poi/poi.js

```diff
@@ -32,23 +34,14 @@ export default class Poi {
       this.type === zoomType.type
     )
     if (zoomSetting) {
-      this.zoom = zoomSetting.zoom
+      return zoomSetting.zoom
     } else {
-      this.zoom = DEFAULT_ZOOM
+      return DEFAULT_ZOOM
     }
   }
 
   poiStoreLiteral() {
-    return {
-      latLon: this.latLon,
-      id: this.id,
-      name: this.name,
-      className: this.className,
-      subClassName: this.subClassName,
-      zoom: this.zoom,
-      type: 'poi',
-      bbox: this.bbox,
-    }
+    return this
```

Comment:
I am not fond of this solution.  
I think it would be useful to keep an explicit list of the keys that will be stored on the client. For example, to make future migrations easier, or prevent serialization inconsistencies between different browsers.

Reply:
I'll fix this in a more élégant way :)  (lodash have i very usefull pick method)